netbox_data_import/templates/netbox_data_import/trace_workspace.html

```
@@ -0,0 +1,281 @@
+        {% csrf_token %}
+        <input type="hidden" name="preview_revision" value="{{ preview_revision }}">
+        <input type="hidden" name="search" id="traceTerminationOfferedSearch">
+        <input type="hidden" name="field_key" id="traceTerminationFieldKey">
+        <input type="hidden" name="object_type" id="traceTerminationObjectType">
+        <input type="hidden" name="object_id" id="traceTerminationObjectId">
+        <div class="modal-header">
+          <h5 class="modal-title">Choose the termination for <span id="traceTerminationLabel"></span></h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        </div>
+        <div class="modal-body">
+          <input type="search" class="form-control mb-2" id="traceTerminationSearch"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)